crates/db_views/community/src/api.rs

```diff
@@ -336,25 +337,30 @@ pub struct ListMultiCommunities {
 
 #[derive(Debug, Serialize, Deserialize, Clone)]
 #[cfg_attr(feature = "ts-rs", derive(ts_rs::TS))]
+#[cfg_attr(feature = "ts-rs", ts(optional_fields, export))]
 pub struct ListMultiCommunitiesResponse {
-  pub multi_communities: Vec<MultiCommunity>,
+  pub multi_communities: Vec<MultiCommunityView>,
 }
 
 #[derive(Debug, Serialize, Deserialize, Clone)]
 #[cfg_attr(feature = "ts-rs", derive(ts_rs::TS))]
+#[cfg_attr(feature = "ts-rs", ts(optional_fields, export))]
 pub struct GetMultiCommunity {
   pub id: MultiCommunityId,
 }
 
 #[derive(Debug, Serialize, Deserialize, Clone)]
 #[cfg_attr(feature = "ts-rs", derive(ts_rs::TS))]
+#[cfg_attr(feature = "ts-rs", ts(optional_fields, export))]
 pub struct GetMultiCommunityResponse {
-  pub multi: MultiCommunity,
+  #[serde(flatten)]
+  pub multi: MultiCommunityView,
```

Comment:
Don't flatten this, it isn't the standard for any of our views. I'd also probably use the verbose `multi_community_view`, just to be explicit, and it matches the other names.

Reply:
Without this the api tests are full of lines like `alphaMulti.multi.multi.name` and `alphaMulti.multi.owner.id` which is very ugly.

Reply:
Usually we use `res` to differentiate them from the views and objectsn within. So:

`alphaMultiRes.multi_community_view.multi_community.id`

Its verbose but clear as to what each sub object is.